salesagility/SuiteCRM
# Feature/collapsed subpanel row count

Adds an optional functionality to indicate whether a collapsed subpanel has any content or not.

![screenshot from 2018-03-23 10-32-43](https://user-images.githubusercontent.com/34056696/37825695-a4eafc3c-2e88-11e8-914d-15d5dc339bff.png)

## Description



When the user enables this functionality, a quick COUNT query will be run to check if the subpanel would contain any rows.

## Motivation and Context

Suggested here #5064 
Preloading every subpanel is slow, but with collapsed panels the user may waste a lot of time expanding empty subpanels. 
This functionality intends to make the life of the user easier by giving a clear visual hint if a subpanel contains records or not.

![screenshot from 2018-03-23 10-32-17](https://user-images.githubusercontent.com/34056696/37825789-f76cbd10-2e88-11e8-889b-a22bd73b3a7d.png)

## How To Test This

Go into the user settings, layout options and enable "Show collapsed subpanel hint"
See any record with subpanels

## Types of changes


### Final checklist
